pymessenger/bot.py

```diff
@@ -368,3 +368,58 @@ def remove_persistent_menu(self):
             )
             result = response.json()
             return result
+        
+    def send_text_quick_replies(self, recipient_id, listOfReplies, messageToReplyTo, listOfPayloads=[]):
+        """
+        Sends a list of text quick replies with an optional message to send before sending the quick replies
+        Payload and Message are optional, however, if no payload for a specific reply (i.e. None) 
+        or for all quick replies, the payload will be defined as the reply itself.
+        
+        
+        https://developers.facebook.com/docs/messenger-platform/send-messages/quick-replies/#text
+        
+        Output:
+            Response from API as <dict>
+        """
+        quickRepliesList = []
+        
+        # If no payloads identified
+        if len(listOfPayloads) == 0:     
+            for reply in listOfReplies:
+                quickRepliesList.append({
+                                        "content_type":"text",
+                                        "title":reply,
+                                        "payload":reply
+                                        })
+            
+        # If payloads is identified
+        else:
+            for reply, payload in zip(listOfReplies, listOfPayloads):
+                # if some payload is not identified in the list
+                if payload == None:
+                    quickRepliesList.append({
+                                            "content_type":"text",
+                                            "title":reply,
+                                            "payload":reply
+                                            })
+                if payload != None:
+                    quickRepliesList.append({
+                                            "content_type":"text",
+                                            "title":reply,
+                                            "payload":payload
+                                            })
+            
+            # if the length of payloads is less than replies, then just let the payload for the rest to be just the reply
+            if len(listOfPayloads) < len(listOfReplies):
+                for reply in listOfReplies[len(listOfPayloads):]:
+                    quickRepliesList.append({
+                                            "content_type":"text",
+                                            "title":reply,
+                                            "payload":reply
+                                            })
+                    
+        return  self.send_message(recipient_id, {
+                "text": message,
```

Comment:
I tried to get this working in a local fork of `Pymessenger`, and I'm encountering an error stating that `message` from line 422 isn't defined.

Reply:
Yeah sorry about that. I fixed the message to be messageToReplyTo. I also added a default message for the function. I tested the function out with whitespace and empty strings, which apparently does not send the quick replies (because it requires a concrete message to reply to).